deepchem/data/data_loader.py

```diff
@@ -164,9 +126,9 @@ def __init__(self,
     self.log_every_n = log_every_n
 
   def featurize(self,
-                inputs: Sequence[Any],
+                inputs: OneOrMany[Any],
                 data_dir: Optional[str] = None,
-                shard_size: Optional[int] = 8192) -> Dataset:
+                shard_size: Optional[int] = None) -> Dataset:
```

Comment:
What does `shard_size = None` mean?  Does that put the entire dataset into one shard?  If so, why change the default value?